src/ai/backend/manager/models/gql_models/agent.py

```diff
@@ -722,11 +722,13 @@ async def batch_load(
         graph_ctx: GraphQueryContext,
         agent_ids: Sequence[AgentId],
         *,
-        domain_name: str | None,
+        access_key: Optional[str] = None,
+        domain_name: Optional[str] = None,
         raw_status: Optional[str] = None,
         scaling_group: Optional[str] = None,
-        access_key: str,
-    ) -> Sequence[Agent | None]:
+    ) -> Sequence[AgentSummary | None]:
+        assert access_key is not None
```

Comment:
@fregataa Existing types of `access_key` and `domain_name` cause the following type error.
So, I changed the types, but please let me know if there’s a better solution.

```
error: Argument 2 to "get_loader_by_func" of "DataLoaderManager" has incompatible type "Callable[[GraphQueryContext, Sequence[AgentId], NamedArg(str | None, 'access_key'), DefaultNamedArg(str | None, 'domain_name'), DefaultNamedArg(str | None, 'raw_status'), DefaultNamedArg(str | None, 'scaling_group')], Coroutine[Any, Any, Sequence[AgentSummary | None]]]"; expected "Callable[[GraphQueryContext, Sequence[AgentId]], Awaitable[Sequence[AgentSummary | None]]]"
```